/zoom/cv_camera_zoom:
  ros__parameters:
    cv_cap_prop_fourcc: 1196444237.0
    device_id: 0
    device_path: 3-4.4.2:1.0
    file_path: ''
    frame_id: camera
    publish_rate: 10.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: best_effort
    read_rate: 30.0
    topic_name: /video_mapping/zoom
    use_sim_time: false

/right/cv_camera_right:
  ros__parameters:
    cv_cap_prop_fourcc: 1196444237.0
    device_id: 0
    device_path: 3-4.4.1:1.0
    file_path: ''
    frame_id: camera
    publish_rate: 10.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: best_effort
    read_rate: 30.0
    topic_name: /video_mapping/right
    use_sim_time: false

/left/cv_camera_left:
  ros__parameters:
    cv_cap_prop_fourcc: 1196444237.0
    device_id: 0
    device_path: 3-4.4.4:1.0
    file_path: ''
    frame_id: camera
    publish_rate: 10.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: best_effort
    read_rate: 30.0
    topic_name: /video_mapping/left
    use_sim_time: false

/back/cv_camera_back:
  ros__parameters:
    cv_cap_prop_fourcc: 1196444237.0
    device_id: 0
    device_path: 3-4.4.3:1.0
    file_path: ''
    frame_id: camera
    publish_rate: 10.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: best_effort
    read_rate: 30.0
    topic_name: /video_mapping/back
    use_sim_time: false

